src/android-service.sh

```diff
@@ -29,7 +30,7 @@ start() {
 	fi
 
 	# Now, wait
-	waitforservice init.svc.${service_service}
+	timeout 10 wait4service init.svc.${service_service}
```

Comment:
should be `waitforservice`. But interesting, on which service you encounter locks? Waiting forever looks the correct way IMHO, as if the service is not running then we can start the depending Droidian service too early

Reply:
Typo fixed.

Will remove timeout, but while debugging stop/kill issue, happened so many times that waitforservice was waiting for a service that was indeed running (because not properly stopped).

Reply:
I think we can keep the timeout, but perhaps we should increase it to a minute or so, and if expires I think the best thing would be to exit 1 so that the service depending on it doesn't start (and perhaps can be useful to the user when looking at systemctl to see the service being marked as failed)

Reply:
Just keep it without the timeout, should be OK and if not, we will add it later :)